library/strudel-synths.scd

```diff
@@ -250,20 +250,22 @@
     //emulation of juno 60 chorus
     SynthDef("strudel_chorus" ++ numChannels, { |out, mix, depth, speed|
 		var input = In.ar(out, numChannels);
-        var l;
-        var r;
+        var l = input[0];
```

Comment:
`input` may have any number of channels, and if someone chooses to run it mono, it will not be an array. You could use the technique we use in `dirt_leslie`.

Reply:
did not know that it was not always an array, Ill fix that!